src/@components/CategoryPage/CategoryContents/index.tsx

```diff
@@ -33,9 +33,12 @@ export default function CategoryContents() {
                 rowStart={gridValue[index].rowStart}
                 rowEnd={gridValue[index].rowEnd}
                 onClick={() => moveCategory(item._id)}
-                imgUrl={item.imgurl}>
-                <St.CategoryTitle>{item.title}</St.CategoryTitle>
-                <St.CategoryDescription>{item.content}</St.CategoryDescription>
+                imgUrl={item.imgurl}
+                className={`GTM_${index}`}>
```

Comment:
`GTM_0` `GTM_1` 은 이후 중복 위험이 크다고 생각듭니다!
아래와 같은 네이밍은 어떨까 합니다

```suggestion
                className={`GTM_category${index}`}>
```